### PR TITLE
MAINT: Misc mypy fixes.

### PR DESCRIPTION
**Pull request by Carreau**
It seem like since about a week mypy check have been failing everywhere.
It does not seem to be a new version of mypy, since the lastest is at
least a month old, but this shoudl fix some of the errors.

In particular mypy wants properties setter to be just next to the
getters.

### CODE REVIEW

File: .github/workflows/test_typing.yml [hunk 1]
```diff
@@ -19,4 +19,5 @@ jobs:
           pip install -e .[all]
 
       - name: Run mypy on typed modules
+        if: ${{ false }}
```
Comment (tlambert03) [inline .github/workflows/test_typing.yml]: is this just skipping them?

Comment (Carreau) [reply .github/workflows/test_typing.yml]: Yes. It just does not run this step. I can also just comment the lines...

Comment (tlambert03) [reply .github/workflows/test_typing.yml]: I guess I'm confused then if this PR is fixing the issues, or just not running type checks anymore

Comment (Carreau) [reply .github/workflows/test_typing.yml]: Ah sorry I probably merged this and #4950

Comment (Carreau) [reply .github/workflows/test_typing.yml]:
```suggestion
```

**Comment by tlambert03** [comment]
The problem here appears to be that, for some reason, running `mypy path/to/folder` is no longer restricting mypy to `path/to/folder`.  For example:

```
╰─❯ mypy napari/settings/
napari/_vispy/visuals/clipping_planes_mixin.py:26: error: Argument 2 for "super" not an instance of argument 1  [misc]
napari/_vispy/visuals/clipping_planes_mixin.py:28: error: "_PVisual" has no attribute "_clip_filter"  [attr-defined]
napari/_vendor/experimental/cachetools/cachetools/abc.py:16: error: "Type[DefaultMapping]" has no attribute "__missing__"  [attr-defined]
napari/utils/mouse_bindings.py:19: error: Function "builtins.callable" is not valid as a type  [valid-type]
napari/utils/mouse_bindings.py:19: note: Perhaps you meant "typing.Callable" instead of "callable"?
napari/utils/mouse_bindings.py:20: error: Function "builtins.callable" is not valid as a type  [valid-type]
napari/utils/mouse_bindings.py:20: note: Perhaps you meant "typing.Callable" instead of "callable"?
napari/utils/mouse_bindings.py:21: error: Function "builtins.callable" is not valid as a type  [valid-type]
napari/utils/mouse_bindings.py:21: note: Perhaps you meant "typing.Callable" instead of "callable"?
napari/utils/mouse_bindings.py:22: error: Function "builtins.callable" is not valid as a type  [valid-type]
...
```

we have never been type checking the whole repo.  the `make typecheck` command runs `mypy napari/settings napari/types.py napari/plugins` ... so since this PR attempts to fix typing in modules that we've never been checking in the first place, I'd say we should close it and figure out why mypy is no longer restricting itself to the appropriate folders (or turn it off until we can figure it out).

I'll gently close this @Carreau :) ... but feel free to reopen if you disagree

**Pull request closed without merging**